mittab/apps/tab/templates/all_tab_cards.html

```diff
@@ -1,13 +1,15 @@
-<link rel="stylesheet" type="text/css" href="/static/css/tab_cards.css" />
-<link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8.17/themes/smoothness/jquery-ui.css" />
+<link rel="stylesheet" type="text/css" href="/static/css/tab_cards.css"/>
+<link rel="stylesheet" type="text/css"
+      href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8.17/themes/smoothness/jquery-ui.css"/>
 <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.6.4/jquery.min.js"></script>
 <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8.16/jquery-ui.min.js"></script>
 <script type="text/javascript" src="/static/javascript/tab.js"></script>
 
 <div id="progressbar" class="printable" data-count="0" data-max="{{ all_teams|length }}"></div>
 
 {% for team in all_teams %}
-<div class="printable tab_card_item {% if forloop.counter|divisibleby:2 %}page_break_after{% endif %}" data-team-id={{team.id}}></div>
-
+    <div id="tab-cards" class="printable tab_card_item {% if forloop.counter|divisibleby:2 %}page_break_after{% endif %}">
```

Comment:
This has an extra ">" bracket which prevents the proper rendering of the divs -> prevents javascript from filling in all_tab_cards.

I believe you want to nix the ">" after the endif